superset-frontend/src/components/FilterableTable/index.tsx

```diff
@@ -188,9 +178,6 @@ const StyledFilterableTable = styled.div`
   `}
 `;
 
-// when more than MAX_COLUMNS_FOR_TABLE are returned, switch from table to grid view
```

Comment:
Can you remove unused properties?
- `headerHeight`
- `overscanColumnCount`
- `overscanRowCount`
- `rowHeight`

and fix the `'getCellContent' was used before it was defined` lint error?